app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx

```diff
@@ -103,9 +102,9 @@ export const ApiConnectorCreatorSelectMethod: React.FunctionComponent<IApiConnec
    */
   const onAddUrlSpecification = (e: React.FormEvent<HTMLInputElement>) => {
     const newUrl = e.currentTarget.value;
-    const fileExt = checkDocStyle(newUrl);
+    const fileExt = formatFileName(newUrl);
 
-    if (fileExt === 'wsdl' || fileExt.includes('?WSDL')) {
+    if (fileExt === 'wsdl' || fileExt.includes('?wsdl')) {
```

Comment:
Wonder if doing `fileExt.toLowerCase()` would be better here, that'd cover all the bases.